### ISSUE TITLE
theme?

### ISSUE THREAD

**Issue by Badcholo** [Greymerk/minecraft-roguelike#40]
Is there a way to add Lycanites mobs into the dungeons? a config or theme like doomlike dungeons has? also is there a way to create a dungeon world where the dungeons spawn very close to each other so i can reset it when needed?

**Comment by Badcholo** [comment]
i'd like to add that the command /roguelike dungeon here gives me an error that "here" is not a number. I changed it to 1 and it gave me an error occurred while trying to perform this command.

**Comment by Greymerk** [comment]
It's probably better if you not bundle two issues into one. But anyway...

first of all, what version is this for?

second of all double check your spelling for the command, it's still working.

Yes, you can customize spawners as described in
https://github.com/Greymerk/minecraft-roguelike/wiki/Spawner-Settings

Spawner settings are a field in level settings
https://github.com/Greymerk/minecraft-roguelike/wiki/Level-Settings

**Comment by Badcholo** [comment]
ok i'll study up on how to. I think i can get something working. I could use an example config to get me started i guess. Anyone make a tutorial video?

**Comment by Greymerk** [comment]
https://github.com/Greymerk/minecraft-roguelike/files/67687/settings-demo.zip

of specific interest is _mobs.json

**Comment by Badcholo** [comment]
Thank you.

**Comment by EnderStar** [comment]
Hi, so if i only want to change the spawners, can i only add _mobs.json to the settings folder?
If i only add the mob names to that file, will mobs spawn with the custom armors and effects that the mod adds as default? If not, where can i find the default nbt that the mod adds, i searched in the source but i didnt find anything.
Thanks!
TLDR: I only want to disable some mobspawners.
